dhall-kubernetes-generator.cabal

```diff
@@ -18,7 +18,6 @@ executable dhall-kubernetes
       Dhall.Kubernetes.Convert
       Dhall.Kubernetes.Data
       Dhall.Kubernetes.Types
-      Paths_dhall_kubernetes
```

Comment:
@arianvp https://www.haskell.org/cabal/users-guide/developing-packages.html#accessing-data-files-from-package-code

Reply:
TL;DR: you want it to refer to static data without using Template Haskell